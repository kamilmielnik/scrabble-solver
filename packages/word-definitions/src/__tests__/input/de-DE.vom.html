
<!DOCTYPE html>
<html lang="de" itemscope itemtype="http://schema.org/WebPage">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">



    <base href="/">

    <title>vom – Schreibung, Definition, Bedeutung, Synonyme, Beispiele | DWDS</title>

    <!-- preloading CSS and fonts -->
    <link href="https://www.dwds.de/dwds_static/fonts/crimson-text-v8-latin-regular.woff2" as="font" rel="preload" crossorigin="anonymous">
    <link href="https://www.dwds.de/dwds_static/fonts/source-sans-pro-v21-latin-ext_latin-italic.woff2" as="font" rel="preload" crossorigin="anonymous">
    <link href="https://www.dwds.de/dwds_static/bootstrap-3.3.7-dist/fonts/glyphicons-halflings-regular.woff2" as="font" rel="preload" crossorigin="anonymous">
    <link href="https://www.dwds.de/dwds_static/fonts.css" rel="preload" as="style" onload="this.rel='stylesheet'" crossorigin="anonymous">
    <link href="https://www.dwds.de/dwds_static/fonts.css" rel="stylesheet preload" as="style" crossorigin="anonymous">

    <!-- feeds -->
    <link rel="search" type="application/opensearchdescription+xml" title="DWDS Suche" href="/search.xml">
    <link rel="alternate" type="application/atom+xml" title="DWDS: Artikel des Tages" href="/api/feed/adt"/>
    <link rel="alternate" type="application/atom+xml" title="DWDS: Neueste Artikel" href="/api/feed/latest"/>
    <!-- //feeds -->

    <meta property="og:title" content="DWDS – Digitales Wörterbuch der deutschen Sprache" />
    <meta name="twitter:title" content="DWDS – Digitales Wörterbuch der deutschen Sprache" />
    <meta itemprop="name" content="DWDS – Digitales Wörterbuch der deutschen Sprache" />


      <meta name="description" content="vom – Schreibung, Definition, Bedeutung, Synonyme, Beispiele | DWDS" />
      <meta property="og:description" content="vom – Schreibung, Definition, Bedeutung, Synonyme, Beispiele | DWDS" />
      <meta name="twitter:description" content="vom – Schreibung, Definition, Bedeutung, Synonyme, Beispiele | DWDS" />
      <link rel="canonical" href="https://www.dwds.de/wb/vom"/>



      <meta property="og:image" content="https://www.dwds.de/dwds_static/images/dwds_601_141.png" />


    <meta property="og:url" content="https://www.dwds.de/wb/vom" />
    <meta property="og:type" content="article" />
    <meta property="og:locale" content="de" />
    <meta property="og:site_name" content="DWDS" />

    <meta name="twitter:card" content="summary_large_image" />


      <meta property="twitter:image:src" content="https://www.dwds.de/dwds_static/images/d-200.png" />


    <meta name="twitter:site" content="@dwds_de"/>


      <meta property="image" content="https://www.dwds.de/dwds_static/images/dwds_601_141.png" />


    <meta itemprop="url" content="https://www.dwds.de/wb/vom" />

    <!-- favicon -->
    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png?v=QEJw5qWp5M">
    <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png?v=QEJw5qWp5M">
    <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png?v=QEJw5qWp5M">
    <link rel="manifest" href="/site.webmanifest?v=QEJw5qWp5M" crossorigin="use-credentials">
    <link rel="mask-icon" href="/safari-pinned-tab.svg?v=QEJw5qWp5M" color="#125993">
    <link rel="shortcut icon" href="/favicon.ico?v=QEJw5qWp5M">
    <meta name="msapplication-TileColor" content="#125993">
    <meta name="theme-color" content="#ffffff">
    <!-- //favicon -->

    <!-- Bootstrap -->
    <link href="https://www.dwds.de/dwds_static/bootstrap-3.3.7-dist/css/bootstrap.min.css" rel="stylesheet">

    <script type="application/ld+json">
  [
    {
      "@context": "https://schema.org",
      "@type": "WebSite",
      "url": "/",
      "potentialAction": {
        "@type": "SearchAction",
        "target": "/?q={search_term_string}",
        "query-input": "required name=search_term_string"
      }
    },
    {
      "@context": "https://schema.org",
      "@type": "Organization",
      "url": "/",
      "logo": "https://www.dwds.de/dwds_static/images/dwds_601_141.png",
      "sameAs" : [
        "https://twitter.com/dwds_de"
      ],
      "contactPoint": {
        "@type": "ContactPoint",
        "telephone": "+4930203700",
        "email": "dwds@bbaw.de"
      }
    }
  ]
</script>



  </head>

  <body>
    <header>
      <noscript>
        <div class="alert alert-danger" style="text-align:center;margin-bottom:5px">
          Um den vollen Funktionsumfang dieser Webseite nutzen zu können, muss JavaScript aktiviert sein.<br />
          <a href="https://www.enable-javascript.com/de" target="_blank">Hier finden Sie Hinweise, wie Sie JavaScript in Ihrem Browser aktivieren können.</a>
        </div>
      </noscript>

      <div id="top"></div>

      <!-- spinner -->
      <div id="spinner" style="display:none">Bitte warten Sie einen Moment &hellip;</div>





      <!-- navbar -->
      <nav class="navbar navbar-default navbar-static-top dwds-top">
        <div class="container-fluid" style="padding-left:0">
          <div class="navbar-header">
            <a class="navbar-brand visible-xs-inline" href="/" style="margin:1px;padding:0 0 0 15px"><img src="https://www.dwds.de/dwds_static/images/dwds.svg" width="210" height="50" alt="DWDS-Logo" /></a>
            <a class="navbar-brand hidden-xs" href="/" style="margin:1px;padding:0"><img src="https://www.dwds.de/dwds_static/images/dwds.svg" width="210" height="50" data-toggle="tooltip" data-placement="right" title="Digitales Wörterbuch der deutschen Sprache" alt="DWDS-Logo" /></a>


              <div class="visible-xs-inline-block navbar-right-mobile">


                    <a href="https://www.dwds.de/profile/login?c=wb%2Fvom" class="btn btn-sm btn-default" rel="nofollow"><span class="glyphicon glyphicon-log-in"></span> Login</a>


              </div>

          </div>
          <div id="navbar" class="navbar-collapse collapse">
            <ul class="nav navbar-nav navbar-left">
              <li class="hidden-xs hidden-sm sf" style="color:#fff; padding-left:15px; padding-top:15px">


                  <span class="dwds-claim">

                  Der deutsche Wortschatz von 1600 bis heute.


                  </span>

              </li>
            </ul>

              <ul class="nav navbar-nav navbar-right hidden-xs">


                    <li>
                      <p class="navbar-btn">
                        <a href="https://www.dwds.de/profile/login?c=wb%2Fvom" class="btn btn-sm btn-default" rel="nofollow"><span class="glyphicon glyphicon-log-in"></span> Anmelden</a>
                      </p>
                    </li>


              </ul>

          </div>
        </div>
      </nav>

      <div class="visible-xs-block visible-sm-block sans">

          <div style="color:#666; font-weight:normal; text-align:center; font-size:14px; margin-top:0; margin-bottom:10px; border-bottom:1px solid #e0e0e0" class="dwds-claim">

                  Der deutsche Wortschatz von 1600 bis heute.


          </div>

      </div>


    </header>

    <main>

        <div class="container-fluid" style="max-width:1400px">




<div class="row">
  <div class="col-md-9 article-leftcol">
    <div class="row" style="margin-top:0px;margin-bottom:20px">
      <div class="col-md-12">

        <ol class="breadcrumb " itemscope itemtype="https://schema.org/BreadcrumbList">
          <li itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem">
            <a href="/" class="intern" itemprop="item"><span itemprop="name">Startseite</span></a>
            <meta itemprop="position" content="1" />
          </li>
          <li itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem">
            <a href="/sitemap" class="intern" itemprop="item"><span itemprop="name">Wörterbuch</span></a>
            <meta itemprop="position" content="2" />
          </li>
          <li class="active" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem">
            <span itemprop="name">vom – Schreibung, Definition, Bedeutung, Synonyme, Beispiele</span>
            <meta itemprop="position" content="3" />
          </li>
        </ol>

        <form method="get" action="/" class="form-horizontal autosubmit">
          <div class="input-group">
            <span class="sr-only">
              <label for="q">Sucheingabe</label>
            </span>
            <input type="search" class="form-control input-lg" placeholder="Suche im Digitalen Wörterbuch" name="q" id="q" value="vom" />
            <input type="hidden" name="from" value="wb" />
            <div class="input-group-btn" style="vertical-align:top">
              <button class="btn btn-primary btn-lg" type="submit" aria-label="suchen"><i class="glyphicon glyphicon-search"></i></button>
              <a class="btn btn-default btn-lg" href="/d/suche"><i class="glyphicon glyphicon-question-sign"></i><span class="sr-only">Hilfe zur Suche</span></a>
            </div>
          </div>
        </form>

      </div>
    </div>



    <div class="row">
      <div class="col-md-12">





  <div class="sidebar-wrapper hidden-xs hidden-sm hidden-md">
    <nav class="sidebar-spy" id="spy-1">
      <ul class="sidebar-nav nav" data-spy="affix" data-offset-top="10" data-offset-bottom="200">
        <li>
          <a href="/wb/vom#top" data-target="#top">Seitenanfang</a>
        </li>

        <li>
          <a href="/wb/vom#wb-1" data-target="#wb-1" data-scroll>Bedeutungen</a>
        </li>



        <li>
          <a href="/wb/vom#ot-1" data-target="#ot-1" data-scroll>Thesaurus</a>
        </li>



      </ul>
    </nav>
  </div>


<div id="start-1"></div>

<div class="dwdswb-artikel"><div class="dwdswb-ft" data-track-content="" data-content-name="DWDSWB Artikel" data-content-piece="Formteil"><h1 class="dwdswb-ft-lemmaansatz"><b>vom</b></h1>
<div class="dwdswb-ft-blocks" style="display:table"><div class="dwdswb-ft-block"><span class="dwdswb-ft-blocklabel serif italic">Grammatik </span><span class="dwdswb-ft-blocktext"><span>Präposition + Artikel</span></span></div><div class="dwdswb-ft-block"><span class="dwdswb-ft-blocklabel serif italic">Aussprache </span><span class="dwdswb-ft-blocktext"><span class="btn-link glyphicon glyphicon-volume-up" style="cursor:pointer;top:4px" onclick="$(this).next('audio').trigger('play')"></span><audio><source type="audio/mpeg" src="https://media.dwds.de/dwds2/audio/001/vom.mp3"><span title="Ihr Browser unterstützt kein Audio-Element.">Fehler</span></audio> </span></div><div class="dwdswb-ft-block"></div><div class="dwdswb-ft-block"><span class="dwdswb-ft-blocklabel serif italic">Wortzerlegung</span><span class="dwdswb-ft-blocktext"> <i class="bi bi-arrow-up-right dwds-aur"></i><a href="/wb/von#1">von<sup>1</sup></a> <i class="bi bi-arrow-up-right dwds-aur"></i><a href="/wb/dem">dem</a></span></div></div></div><div class="dwdswb-lesarten" data-track-content="" data-content-name="DWDSWB Artikel" data-content-piece="Bedeutungsteil"><div class="dwdswb-quelle"><abbr data-toggle="tooltip" title="Artikel des Wörterbuchs der deutschen Gegenwartssprache, elektronische Version, 1977">eWDG</abbr></div><h2 id="wb-1">Bedeutung</h2><div class="dwdswb-lesart" id="d-1-1"><div class="dwdswb-lesart-n"><div class="glyphicon glyphicon-chevron-down" style="opacity:.5;font-size:14px"></div></div><div class="dwdswb-lesart-content" style="padding-top:4px"><div class="dwdswb-lesart-def"><span class="dwdswb-verweise"><span class="dwdswb-verweis" data-lemma="von" data-hidx="1" data-lesart="I, II, III 2, IV, V 1, V 3, V 4, V 5, V 6, VI" data-popover-def="1" data-content='&lt;span style="color:#000;font-style:normal"&gt;gibt an, dass ein Ort, eine Person Ausgangspunkt einer Bewegung oder eines Vorgangs ist, der einer Bewegung ähnelt; drückt den Vorgang des Entfernens, den Zustand des Getrenntseins von etw., jmdm. aus; gibt an, dass ein Ort, eine Person Anfangspunkt einer räumlichen Erstreckung, Bezugspunkt für die Kennzeichnung einer Entfernung oder eines Lageverhältnisses ist; gibt die Herkunft, den Ursprung an; gibt das Mittel an; mit; gibt den Bereich an, für den eine Aussage gilt; was … betrifft, hinsichtlich; gibt bei Aussagen über bestimmte Eigenschaften, Verhaltensweisen die Person an, für die die Aussage gilt; gibt den Gegenstand, Inhalt an; über&lt;/span&gt;' data-html="true" data-toggle="popover" data-trigger="hover">siehe auch <a class="intern" href="/wb/von#d-1-I,-II,-III-2,-IV,-V-1,-V-3,-V-4,-V-5,-V-6,-VI">von¹<span class="dwdswb-la"> (I, II, III 2, IV, V 1, V 3, V 4, V 5, V 6, VI)</span></a></span></span></div><div class="dwdswb-verwendungsbeispiele"><div class="dwdswb-block-label">Beispiele:</div><div class="dwdswb-kompetenzbeispiel"><span class="dwdswb-belegtext">sie stammt <span class="dwdswb-stichwort">vom</span> Lande</span></div><div class="dwdswb-kompetenzbeispiel"><span class="dwdswb-belegtext">er durchsucht das Haus <span class="dwdswb-stichwort">vom</span> Keller bis zum Boden</span></div><div class="dwdswb-kompetenzbeispiel"><span class="dwdswb-belegtext"><abbr title="jemanden" data-toggle="tooltip">jmdn.</abbr>, <abbr title="etwas" data-toggle="tooltip">etw.</abbr> nur <span class="dwdswb-stichwort">vom</span> Hörensagen kennen</span></div><button type="button" class="btn btn-default btn-xs more-button expansion-button" style="margin-left:5px;margin-top:-10px" onclick="showExamples('#example-1idm24example')" id="example-1idm24example-button-show"><span style="color:#000">...</span> 3 weitere Beispiele</button><div id="example-1idm24example" class="more-block" data-id="1idm24example"><div class="dwdswb-kompetenzbeispiel"><span class="dwdswb-diasystematik" style="margin-right:10px"><span class="dwdswb-stilebene">umgangssprachlich</span></span><span class="dwdswb-belegtext">die Arbeit kam nicht <span class="dwdswb-stichwort">vom</span> Fleck</span></div><div class="dwdswb-kompetenzbeispiel"><span class="dwdswb-belegtext"> <span class="dwdswb-stichwort">vom</span> Hundertsten ins Tausendste kommen</span></div><div class="dwdswb-kompetenzbeispiel"><span class="dwdswb-diasystematik" style="margin-right:10px"><span class="dwdswb-stilebene">salopp</span></span><span class="dwdswb-belegtext"><abbr title="jemandem" data-toggle="tooltip">jmdm.</abbr> die Haare <span class="dwdswb-stichwort">vom</span> Kopf fressen</span></div><button type="button" class="btn btn-default btn-xs more-button collapse-button" onclick="showExamples('#example-1idm24example')" id="example-1idm24example-button-hide" style="display:none;margin-left:5px;margin-top:-10px"><span style="color:#000">...</span> weniger Beispiele</button></div></div></div></div></div></div>




<span class="hyphinput" data-id="vom-1" style="display:none">vom</span>






  <div style="clear:both;float:none;margin-top:40px"></div>
  <div class="etymwb-quelle"><a href="https://www.openthesaurus.de/">www.openthesaurus.de</a> (<time datetime="2022-11">11/2022</time>)</div>
  <h2 id="ot-1">Thesaurus</h2>
  <div class="row" style="margin-bottom:10px">
    <div class="col-md-12" data-track-content data-content-name="DWDSWB Artikel" data-content-piece="Thesaurus">

  <div style="padding:3px 3px 15px 3px; margin-left:3em">



      <div style="position:relative">

        <div class="serif italic" style="font-size:12pt;font-weight:normal">Synonymgruppe</div>
        <div class="sans" style="font-size:16pt">
          vom &middot;
          von dem

        </div>
      </div>
















  </div>


    </div>
  </div>










  <script type="application/ld+json">
    [
   {
      "@context" : "http://schema.org/",
      "@type" : "WebPage",
      "inlanguage" : "de",
      "isPartOf" : "dwds.de",
      "url" : "https://www.dwds.de/wb/vom"
   },
   {
      "@context" : "https://schema.org",
      "@type" : "BreadcrumbList",
      "itemListElement" : [
         {
            "@type" : "ListItem",
            "item" : {
               "@id" : "https://www.dwds.de/",
               "name" : "Startseite"
            },
            "position" : 1
         },
         {
            "@type" : "ListItem",
            "item" : {
               "@id" : "https://www.dwds.de/wb/vom",
               "name" : "Wortinformation zu „vom“"
            },
            "position" : 2
         }
      ]
   },
   {
      "@context" : "https://schema.org",
      "@type" : "AudioObject",
      "caption" : "Aussprache vom",
      "contentUrl" : "https://media.dwds.de/dwds2/audio/001/vom.mp3",
      "encodingFormat" : "audio/mpeg",
      "name" : "vom.mp3"
   }
]

  </script>







            <div class="citation-help">
              <span class="label label-default">Zitationshilfe</span>
              <blockquote>
                <cite>
                  „vom“, bereitgestellt durch das Digitale Wörterbuch
                  der deutschen Sprache, &lt;https://www.dwds.de/wb/vom&gt;<span class="today-cit"></span>.
                </cite>
                <p><a href="/d/zitieren">Weitere Informationen &hellip;</a></p>
              </blockquote>
            </div>




          <table class="table table-condensed sans">
            <tr>
              <th style="border:none">alphabetisch vorangehend</th>
              <th style="border:none;text-align:right">alphabetisch nachfolgend</th>
            </tr>
            <tr>
              <td>

                  <a href="/wb/volvieren">volvieren</a><br/>

                  <a href="/wb/voluptu%C3%B6s">voluptuös</a><br/>

                  <a href="/wb/voluntativ">voluntativ</a><br/>

                  <a href="/wb/voluntaristisch">voluntaristisch</a><br/>

                  <a href="/wb/volumin%C3%B6s">voluminös</a>

              </td>
              <td style="text-align:right">

                  <a href="/wb/vom%20Dienst">vom Dienst</a><br/>

                  <a href="/wb/vom%20Fleck%20weg">vom Fleck weg</a><br/>

                  <a href="/wb/vom%20Fleisch%20fallen">vom Fleisch fallen</a><br/>

                  <a href="/wb/vom%20Fleische%20fallen">vom Fleische fallen</a><br/>

                  <a href="/wb/vom%20Hocker%20fallen">vom Hocker fallen</a>

              </td>
            </tr>
          </table>


      </div>
    </div>
  </div>

  <div class="col-md-3 rightcol">
    <div>


          <p class="h3">Worthäufigkeit <sup><a href="/d/worthaeufigkeit"><i class="glyphicon glyphicon-question-sign" style="opacity:.5" title="Dokumentation zu Worthäufigkeiten im DWDS" data-toggle="tooltip" data-placement="bottom"></i></a></sup></p>


            <table class="serif italic word-frequency">
              <tr>
                <td>selten</td>
                <td></td>
                <td></td>
                <td></td>
                <td></td>
                <td></td>
                <td style="text-align:right">häufig</td>
              </tr>
              <tr>

                  <td style="text-align:center">

                      <div class="word-frequency-active"></div>

                  </td>

                  <td style="text-align:center">

                      <div class="word-frequency-active"></div>

                  </td>

                  <td style="text-align:center">

                      <div class="word-frequency-active"></div>

                  </td>

                  <td style="text-align:center">

                      <div class="word-frequency-active"></div>

                  </td>

                  <td style="text-align:center">

                      <div class="word-frequency-active"></div>

                  </td>

                  <td style="text-align:center">

                      <div class="word-frequency-active"></div>

                  </td>

                  <td style="text-align:center">

                      <div class="word-frequency-inactive"></div>

                  </td>

              </tr>
            </table>




            <div id="plot-box" style="font-size:smaller; margin-top:5px; text-align:left" data-track-content data-content-name="DWDSWB Artikel" data-content-piece="Verlaufskurve">
              <p class="h3">Wortverlaufskurve <sup><a href="/d/plot"><i class="glyphicon glyphicon-question-sign" style="opacity:.5" title="Dokumentation zu den DWDS-Wortverlaufskurven" data-toggle="tooltip" data-placement="bottom"></i></a></sup></p>
              <ul class="nav nav-tabs" role="tablist" id="plot-tabs" style="bottom:-20px;position:relative;margin-top:-20px;padding-left:0">
                <li role="presentation">
                  <a href="https://www.dwds.de/wb/vom#plot-1" aria-controls="plot-1" role="tab" data-toggle="tab" style="padding:5px 15px">ab 1600</a>
                </li>
                <li role="presentation" class="active">
                  <a href="https://www.dwds.de/wb/vom#plot-2" aria-controls="plot-2" role="tab" data-toggle="tab" style="padding:5px 15px">ab 1946</a>
                </li>
              </ul>
              <div style="float:none;clear:both"></div>

                <div class="tab-content">
                  <div role="tabpanel" class="tab-pane" id="plot-1">
                    <a href="https://www.dwds.de/r/plot/?xrange=1600:1999&amp;window=10&amp;slice=3&amp;q=vom&amp;corpus=dta%2Bdwds"><img src="/r/plot/image/?v=hist&amp;q=vom" style="width:100%" alt="Wortverlaufskurve 1600−1999" title="Wortverlaufskurve 1600−1999" data-toggle="tooltip" data-placement="bottom" width="300" height="180" loading="lazy" /></a>
                  </div>
                  <div role="tabpanel" class="tab-pane active" id="plot-2">
                    <a href="https://www.dwds.de/r/plot/?xrange=1946:2022&amp;window=3&amp;slice=1&amp;q=vom&amp;corpus=zeitungenxl"><img src="/r/plot/image/?v=pres&amp;q=vom" style="width:100%" alt="Wortverlaufskurve ab 1946" title="Wortverlaufskurve ab 1946" data-toggle="tooltip" data-placement="bottom" width="300" height="180" /></a>
                  </div>
                </div>

            </div>







    </div>



    <p class="h3">Weitere Wörterbücher</p>
    <ul class="sans" data-track-content data-content-name="DWDSWB Artikel" data-content-piece="Links zu weiteren Wörterbüchern">
      <li>
        <a href="/wb/dwb/search/?q=vom">Deutsches Wörterbuch (¹DWB)</a>
      </li>
      <li>
        <a href="/wb/dwb2/search/?q=vom">Deutsches Wörterbuch, Neubearbeitung (²DWB)</a>
      </li>
      <li><a href="/wb/wdg/search/?q=vom">Wörterbuch der deutschen Gegenwartssprache (WDG)</a></li>


    </ul>

    <p class="h3">Belege in Korpora</p>

    <div data-track-content data-content-name="DWDSWB Artikel" data-content-piece="Links zu Korpora">



        <p class="h4">Referenzkorpora</p>
        <ul class="sans"><li class="" data-corpus="kern" data-query="vom" data-lexdb="1" >
              <a href="https://www.dwds.de/r/?corpus=kern&amp;q=vom">DWDS-Kernkorpus (1900–1999)</a></li>
          <li class="" data-corpus="korpus21" data-query="vom" data-lexdb="1" >
              <a href="https://www.dwds.de/r/?corpus=korpus21&amp;q=vom">DWDS-Kernkorpus 21 (2000–2010)</a></li>
          <li class="" data-corpus="dtak" data-query="vom" data-lexdb="1" >
              <a href="https://www.dwds.de/r/?corpus=dtak&amp;q=vom">DTA-Kernkorpus (1598–1913)</a></li>

        </ul>

        <p class="h4">Metakorpora</p>
        <ul class="sans"><li class="" data-corpus="dta" data-query="vom" data-lexdb="1" >
              <a href="https://www.dwds.de/r/?corpus=dta&amp;q=vom">DTA-Kern+Erweit. (1465–1969)</a></li>
          <li class="" data-corpus="dtaxl" data-query="vom" data-lexdb="1" >
              <a href="https://www.dwds.de/r/?corpus=dtaxl&amp;q=vom">Historische Korpora (1465–1998)</a></li>
          <li class="" data-corpus="public" data-query="vom" data-lexdb="1" >
              <a href="https://www.dwds.de/r/?corpus=public&amp;q=vom">Referenz- und Zeitungskorpora (frei)</a></li>

        </ul>

        <p class="h4">Zeitungskorpora</p>
        <ul class="sans"><li class="" data-corpus="bz" data-query="vom" data-lexdb="1" >
              <a href="https://www.dwds.de/r/?corpus=bz&amp;q=vom">Berliner Zeitung (1994–2005)</a></li>
          <li class="" data-corpus="tsp" data-query="vom" data-lexdb="1" >
              <a href="https://www.dwds.de/r/?corpus=tsp&amp;q=vom">Der Tagesspiegel (ab 1996)</a></li>
          <li class="" data-corpus="zeit" data-query="vom" data-lexdb="1" >
              <a href="https://www.dwds.de/r/?corpus=zeit&amp;q=vom">Die ZEIT (1946–2018)</a></li>

        </ul>

        <p class="h4">Webkorpora</p>
        <ul class="sans"><li class="" data-corpus="blogs" data-query="vom" data-lexdb="1" >
              <a href="https://www.dwds.de/r/?corpus=blogs&amp;q=vom">Blogs</a></li>

        </ul>

        <p class="h4">Spezialkorpora</p>
        <ul class="sans"><li class="" data-corpus="dtae" data-query="vom" data-lexdb="1" >
              <a href="https://www.dwds.de/r/?corpus=dtae&amp;q=vom">DTA-Erweiterungen (1465–1969)</a></li>
          <li class="" data-corpus="adg" data-query="vom" data-lexdb="1" >
              <a href="https://www.dwds.de/r/?corpus=adg&amp;q=vom">Archiv der Gegenwart (1931–2000)</a></li>
          <li class="" data-corpus="dingler" data-query="vom" data-lexdb="1" >
              <a href="https://www.dwds.de/r/?corpus=dingler&amp;q=vom">Polytechnisches Journal</a></li>
          <li class="" data-corpus="untertitel" data-query="vom" data-lexdb="1" >
              <a href="https://www.dwds.de/r/?corpus=untertitel&amp;q=vom">Filmuntertitel</a></li>
          <li class="" data-corpus="spk" data-query="vom" data-lexdb="1" >
              <a href="https://www.dwds.de/r/?corpus=spk&amp;q=vom">Gesprochene Sprache</a></li>
          <li class="" data-corpus="ddr" data-query="vom" data-lexdb="1" >
              <a href="https://www.dwds.de/r/?corpus=ddr&amp;q=vom">DDR</a></li>
          <li class="" data-corpus="politische_reden" data-query="vom" data-lexdb="1" >
              <a href="https://www.dwds.de/r/?corpus=politische_reden&amp;q=vom">Politische Reden (1982–2020)</a></li>
          <li class="" data-corpus="bundestag" data-query="vom" data-lexdb="1" >
              <a href="https://www.dwds.de/r/?corpus=bundestag&amp;q=vom">Bundestagskorpus (1949–2017)</a></li>
          <li class="" data-corpus="avh-bern" data-query="vom" data-lexdb="1" >
              <a href="https://www.dwds.de/r/?corpus=avh-bern&amp;q=vom">A. v. Humboldts Publizistik (dt., 1790–1859)</a></li>
          <li class="" data-corpus="bruedergemeine" data-query="vom" data-lexdb="1" >
              <a href="https://www.dwds.de/r/?corpus=bruedergemeine&amp;q=vom">Nachrichten aus der Brüdergemeine (1819–1894)</a></li>
          <li class="" data-corpus="pitaval" data-query="vom" data-lexdb="1" >
              <a href="https://www.dwds.de/r/?corpus=pitaval&amp;q=vom">Der Neue Pitaval (1842–1890)</a></li>
          <li class="" data-corpus="jean_paul" data-query="vom" data-lexdb="1" >
              <a href="https://www.dwds.de/r/?corpus=jean_paul&amp;q=vom">Briefe von Jean Paul (1780–1825)</a></li>
          <li class="" data-corpus="wikipedia" data-query="vom" data-lexdb="1" >
              <a href="https://www.dwds.de/r/?corpus=wikipedia&amp;q=vom">Wikipedia-Korpus</a></li>

        </ul>

    </div>




  </div>
</div>

        </div>

    </main>

    <footer class="dwds-bottom dwds-bottom-box">




      <div class="container">
        <div class="row dwds-bottom-claim">
          <div class="col-md-12">
            DWDS – Digitales Wörterbuch der deutschen Sprache

          </div>
        </div>
        <div class="row">
          <div class="col-md-12">
            <div class="dwds-bottom-box">
              <div class="dwds-bottom-flex">
                <a href="/d/ueber-uns">Über uns</a>
              </div>
              <div class="dwds-bottom-flex">
                <a href="https://www.dwds.de/b">Blog</a>
              </div>
              <div class="dwds-bottom-flex">
                <a href="/d/suche">Hilfe zur Suche</a>
              </div>
              <div class="dwds-bottom-flex">
                <a href="/d/datenschutz">Datenschutz</a>
              </div>
              <div class="dwds-bottom-flex">
                <a href="/d/nutzungsbedingungen">Nutzungsbedingungen</a>
              </div>
              <div class="dwds-bottom-flex">
                <a href="/d/zitieren">Zitieren des DWDS</a>
              </div>
              <div class="dwds-bottom-flex">
                <a href="https://www.dwds.de/kontakt">Kontakt und Feedback</a>
              </div>
              <div class="dwds-bottom-flex">
                <a href="/d/impressum">Impressum</a>
              </div>
              <div class="dwds-bottom-flex">
                <a href="https://www.bbaw.de/"><img src="https://www.dwds.de/dwds_static/images/bbaw.svg" style="height:80px" height="80" width="180" alt="BBAW-Logo" loading="lazy" /></a>
              </div>
            </div>
          </div>
        </div>
      </div>
    </footer>

    <!-- Bootstrap slider, https://github.com/seiyria/bootstrap-slider -->
    <link href="https://www.dwds.de/dwds_static/bootstrap-slider/css/bootstrap-slider.min.css" rel="stylesheet">

    <!--[if lt IE 9]>
      <script src="https://www.dwds.de/dwds_static/js/html5shiv.min.js"></script>
      <script src="https://www.dwds.de/dwds_static/js/respond.min.js"></script>
    <![endif]-->

    <link href="https://www.dwds.de/dwds_static/dwds.min.css?v=19" rel="stylesheet">
    <script>
      base = '/';
      base_static = 'https://www.dwds.de/dwds_static/';
      q    = "vom";

        ansatz = "vom";

      is_lex = 0
      pos = 'Präposition + Artikel';
      dwds_host_staging = 'https://www.dwds.de/';
      dwds_host_dynamic = 'https://www.dwds.de/';
      dwds_host_api     = 'https://www.dwds.de/';

    </script>
    <script src="https://www.dwds.de/dwds_static/dwds.min.js?v=14"></script>

    <script>
      new ClipboardJS('.ddc-copy-hit');
    </script>










      <!-- Matomo -->
      <script>
        var _paq = window._paq = window._paq || [];
        /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
        _paq.push(['trackPageView']);
        _paq.push(['enableLinkTracking']);
        (function() {
          var u="https://stats.dwds.de/";
          _paq.push(['setTrackerUrl', u+'matomo.php']);
          _paq.push(['setSiteId', '1']);
          _paq.push(['enableLinkTracking']);
          _paq.push(['trackPageView']);
          _paq.push(['trackVisibleContentImpressions']);
          var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
          g.async=true; g.src=u+'matomo.js'; s.parentNode.insertBefore(g,s);
        })();
      </script>
      <noscript><p><img src="https://stats.dwds.de/matomo.php?idsite=1&amp;rec=1" style="border:0;" alt="" /></p></noscript>
      <!-- End Matomo Code -->

  </body>
</html>
