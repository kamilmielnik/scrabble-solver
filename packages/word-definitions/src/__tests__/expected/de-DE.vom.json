{
  "definitions": [
    "gibt an, dass ein Ort, eine Person Ausgangspunkt einer Bewegung oder eines Vorgangs ist, der einer Bewegung ähnelt",
    "drückt den Vorgang des Entfernens, den Zustand des Getrenntseins von etw., jmdm. aus",
    "gibt an, dass ein Ort, eine Person Anfangspunkt einer räumlichen Erstreckung, Bezugspunkt für die Kennzeichnung einer Entfernung oder eines Lageverhältnisses ist",
    "gibt die Herkunft, den Ursprung an",
    "gibt das Mittel an",
    "mit",
    "gibt den Bereich an, für den eine Aussage gilt",
    "was … betrifft, hinsichtlich",
    "gibt bei Aussagen über bestimmte Eigenschaften, Verhaltensweisen die Person an, für die die Aussage gilt",
    "gibt den Gegenstand, Inhalt an",
    "über"
  ],
  "exists": true
}
